examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -70,7 +70,7 @@
     "\n",
```

Comment:
<p>Why do you remove the tags.Target from purchase_id_first? that is the only difference between target_feature and groupby_features['purchase_id_first']</p><p>If you want this change, can we delete following code :</p><pre class="ql-syntax" spellcheck="false">target_feature = (
&nbsp; &nbsp; groupby_features['purchase_id_first'] &gt;&gt;&nbsp;
&nbsp; &nbsp; AddMetadata(tags=[Tags.TARGET])
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/849/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>thanks. yes, I think we can remove the code above you pointed put. the reason I removed it because of the transform in the dataloader. I was told  <code>ToTarget</code> method transform column to target to tag is not needed. </p>